.github/workflows/ci.yml

```diff
@@ -36,6 +36,23 @@ jobs:
       - name: Lint
         run: just lint
 
+  msrv:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest] # TODO add back windows-latest https://github.com/TBD54566975/web5-rs/issues/189
```

Comment:
it's probably safe to assume if the `cargo msrv` command passes on ubuntu then it ill pass on all systems, what do ya think?

Reply:
Updated